### DISCUSSION TITLE
How do you print api request response bodies?

### DISCUSSION THREAD

**Discussion by kevinlinxc** [grafana/grafana#68937] [Data Sources]
I am using the grafana-trino plugin and when I add a datasource and test the connection, I get a 400 error:

```
level=info msg="Request Completed" method=POST path=/api/ds/query status=400 remote_addr=[::1] time_ms=1522 duration=1.5226199s si3-67cc-44a2-aa2d-5a8b1b7238a6 handler=/api/ds/query
```

Is there a way to see the body of the response, so that I could better diagnose why I get a 400? Or is there any better way to see what is happening behind the scenes? I am already using a trace log level.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.

**Comment by ALL-SPACE-Anas** [comment]
Hi @kevinlinxc, I'm basically facing the same issue on my kubernetes installation. Did you manage to fix this issue? If yes, can you share how?

**Comment by kevinlinxc** [comment]
@manas-suleman I abandoned the grafana-trino plugin altogether lol. Used a generic API plugin and made my own API.
